crates/ruff/src/commands/rule.rs

```diff
@@ -18,26 +19,25 @@ struct Explanation<'a> {
     linter: &'a str,
     summary: &'a str,
     message_formats: &'a [&'a str],
-    fix: String,
     #[expect(clippy::struct_field_names)]
     explanation: Option<&'a str>,
-    preview: bool,
+    fix: FixAvailability,
+    group: RuleGroup,
```

Comment:
I suggest renaming this to `status`. It might otherwise be mistaken for a rule category (something we want to introduce in the future).